### PR TITLE
TOPIC page 1170: 'Download current page' PDF should have A4 paper size

### PR DESCRIPTION
**Pull request by yuliia-biletska**

### CODE REVIEW

File: plugins/rocks.xml.webhelp/js/xml.rocks.js [hunk 1]
```diff
@@ -47,43 +47,30 @@ function expandCollapseAll() {
 }
 
 //Create PDF from HTML topic content
-function getPDF() {
-    var idTopicArticle = document.getElementById("topic-article");
-    var removedChild = idTopicArticle.querySelector('.related-links');
+var idTopicArticle = document.getElementById("topic-article");
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/js/xml.rocks.js]: I suppose these vars should be moved to the function.

File: plugins/rocks.xml.webhelp/xsl/topic.xsl [hunk 2]
```diff
@@ -199,7 +199,7 @@
                                     </button>
 
                                     <div id="menu-dropdown-download" class="dropdown-content-download">
-                                        <button id="download-page-btn" onclick="getPDF()">
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/topic.xsl:202]: Shouldn't we have 'onclick="ExportPdf()"' here as well?

File: plugins/rocks.xml.webhelp/js/xml.rocks.js [hunk 3]
```diff
@@ -47,43 +47,30 @@ function expandCollapseAll() {
...
-
-        for (var i = 1; i <= totalPDFPages; i++) {
-            pdf.addPage(pdfWidth, pdfHeight);
-            pdf.addImage(imgData, 'JPG', topLeftMargin, -(pdfHeight * i) + (topLeftMargin * 4), canvasImageWidth, canvasImageHeight);
-        }
-
-        pdf.save(topicName + ".pdf");
-    });
-    // Append 'related-links' to the page after downloading current topic
+function ExportPdf() {
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/js/xml.rocks.js]: Function name should start from the lowercase

File: plugins/rocks.xml.webhelp/js/xml.rocks.js [hunk 4]
```diff
@@ -47,43 +47,30 @@ function expandCollapseAll() {
...
+    kendo.drawing
+        .drawDOM("#topic-article",
+            {
+                paperSize: "A4",
+                margin: {top: "1cm", bottom: "1cm", right: "1cm", left: "1cm"},
+                scale: 0.8,
+                height: 500
+            })
+        .then(function (group) {
+            kendo.drawing.pdf.saveAs(group, topicName + ".pdf");
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/js/xml.rocks.js:68]: Looks like topicName is incorrect. I get 'undefined.pdf' filename

Comment (ibarinova) [reply plugins/rocks.xml.webhelp/js/xml.rocks.js:68]: Also, we get console log errors:
image

**Review by vasyl-krupa — APPROVED** [review]
Looks good

**Review by ibarinova — APPROVED** [review]
Approved

**Pull request merged**